concepts/objects/about.md

```diff
@@ -0,0 +1,251 @@
+# About
+
+## Explanation
+
+Besides primitive data types like `number` and `string`, there is another important data type in JavaScript called `object`.
+Objects are collections of key-value pairs.
+In other languages they are referred to as maps or dictionaries and often times values need to have the same data type.
+In JavaScript, only the type of the key is restricted, it has to be a string.
+The values inside one object can have different types.
+They can be primitive types like numbers but also arrays, other objects or even functions.
+
+## Creating an Object
+
+You create an object (literal) with curly brackets.
```

Comment:
Does the student know what `(literal)` refers to at this point in the track?

If not, perhaps adding a brief footnote or explanation would prevent some web searching.

Reply:
Good point. Jeremy recommended to use links in these cases where we use terms that someone that already knows programming should know but in reality might not know.